x-pack/legacy/plugins/apm/public/components/shared/KueryBar/index.tsx

```diff
@@ -179,42 +149,26 @@ export function KueryBar() {
   return (
     <Container>
       <Typeahead
-        disabled={indexPatternMissing}
         isLoading={state.isLoadingSuggestions}
         initialValue={urlParams.kuery}
         onChange={onChange}
         onSubmit={onSubmit}
         suggestions={state.suggestions}
-        queryExample={queryExample}
+        placeholder={i18n.translate('xpack.apm.kueryBar.placeholder', {
+          defaultMessage: `Search {numProcessorEvents, plural,
```

Comment:
I wrote this under the assumption that we might want to filter for instance both transactions and spans, or transactions and metrics, but we're not using it right now, so could be a little simpler perhaps as well with `select`.